File: C&C++/SimarKatyal_Day3/.c
//Q1 : Write a program to calculate the power of a number using the pow() function from the <cmath> library.
#include <stdio.h>
#include <math.h>
int main()
{
    printf("welcome to POWER CHECKER machine!!\n");
    float base, power, result;
    printf("Please enter the base value : ");
    scanf("%f",&base);
    printf("Please enter the power value : ");
    scanf("%f",&power);
    result = pow(base,power);
    printf("%.2f rasied to the power of %.2f is %.2f.\n",base,power,result);
    printf("\nThank you!!\nKeep visiting us!!");

    return 0;
}



//Q2 : Write a program to calculate the natural logarithm (base e) of a given number using the log() function from the <cmath> library.
#include <stdio.h>
#include <math.h>
int main()
{
    printf("welcome to LOGARITHM CHECKER machine!!\n");
    float base, result;
    printf("Please enter the base value : ");
    scanf("%f",&base);
    result = log(base);
    printf("The natural logarithm of %f is %.2f.\n",base,result);
    printf("\nThank you!!\nKeep visiting us!!");

    return 0;
}

File: Python/SimarKatyal_Day2/.py
#Q1 : Write a Python program that takes an integer as input and determines whether it is even or odd.
print("Welcome to EVEN-ODD Machine!!")
try :
  num = int(input("Please enter any integer : "))
except :
  print("Please enter an integer value. ")
else :
  if num==0 :
    print("{} is neither even nor odd.".format(num))
  elif num%2==0 :
    print("{} is an even number.".format(num))
  else :
    print("{} is an odd number.".format(num))
finally :
  print("\nThank you!!\nKeep visiting!!")





#Q2 : Write a Python program that prompts the user to enter a sentence and counts the number of words in the sentence.
#Q1 : Write a Python program that prompts the user to enter a sentence and counts the number of words in the sentence.
print("Welcome to COUNT machine!!")
text = input("Please enter the text : ")
#print(text)
words = len(text.split())
print("Number of words in the entered text : {}".format(words))
print("\nThank you!!\nKeep visiting!!")
